steward/src/cellars.rs

```diff
@@ -40,11 +40,11 @@ pub const ALLOWED_CACHE_PRICE_ROUTER: [&str; 1] = [CELLAR_RYETH];
 // permissions
 
 pub const ALLOWED_V2_0_SETUP_ADAPTORS: [(&str, &str); 1] = [(CELLAR_RYUSD, ADAPTOR_CELLAR_V2)];
-pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 0] = [];
-pub const ALLOWED_V2_5_CATALOGUE_ADAPTORS: [(&str, &str); 2] = [
-    (CELLAR_TURBO_GHO, ADAPTOR_VESTING_SIMPLE_V2),
-    (CELLAR_TURBO_SWETH, ADAPTOR_VESTING_SIMPLE_V2),
+pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 2] = [
```

Comment:
Why is the length of this vec 2? Feels like that shouldn't even compile.

Reply:
fixed